### PR TITLE
fixed spelling, added spelling action

### PR DESCRIPTION
**Pull request by brtnfld**

### CODE REVIEW

File: clinic/index.org [hunk 1]
```diff
@@ -3791,7 +3791,7 @@ export HDF5_USE_FILE_LOCKING="FALSE"
 - "Partial I/O gets in the way" - sorting fields by name or offset
   - Happens on each write call => overhead
 - Can this be avoided? User might provide a patch...
-**** [[https://forum.hdfgroup.org/t/read-write-specific-coordiantes-in-multi-dimensional-dataset/9137][Read/write specific coordiantes in multi-dimensional dataset?]]
+**** [[https://forum.hdfgroup.org/t/read-write-specific-coordinates-in-multi-dimensional-dataset/9137][Read/write specific coordinates in multi-dimensional dataset?]]
```
Comment (loricooperhdf) [inline clinic/index.org:3794]: Do not fix the typo in this URL, at least.

File: documentation/hdf5-docs/release_specifics/sw_changes_1.8.md [hunk 2]
```diff
@@ -2079,7 +2079,7 @@ The patch that was required for Cygwin with HDF5 Release 1.8.0 has been rolled i
 
 HDF5 Release 1.8.0 is a major release with several new features and a variety of bugfixes, optimizations, and portability enhancements.
 
-See the document [Whats New in HDF5 1.8.0](Whats New in HDF5 1.8.0) for a high-level overview of the new functionality in this release. The following pages assume a familiarity with that overview.
+See the document [What's New in HDF5 1.8.0](What's New in HDF5 1.8.0) for a high-level overview of the new functionality in this release. The following pages assume a familiarity with that overview.
```
Comment (loricooperhdf) [inline documentation/hdf5-docs/release_specifics/sw_changes_1.8.md:2082]: This linked document doesn't exist and I don't think we should spend time tracking it down. In my opinion re-write the line like this:

See the document What's New in HDF5 1.8.0 for a high-level

(Remove link)

**Review by loricooperhdf — CHANGES_REQUESTED** [review]

File: clinic/index.org (hunk 1 above)
Comment (brtnfld) [reply clinic/index.org:3794]: The corrected link seems to work though, strange.

File: documentation/hdf5-docs/release_specifics/sw_changes_1.8.md (hunk 2 above)
Comment (brtnfld) [reply documentation/hdf5-docs/release_specifics/sw_changes_1.8.md]: fixed.

File: clinic/index.org (hunk 1 above)
Comment (brtnfld) [reply clinic/index.org:3794]: I corrected the forum post title.

**Review by lkurz — APPROVED** [review]

**Comment by lkurz** [comment]
Thank you Scot for the spell check.

**Pull request merged**